Frontend/speedcart-react/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -2,104 +2,111 @@ import React, { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
 
 import { FaTrash, FaEdit, FaShare, FaClipboard } from 'react-icons/fa';
-import { useAuth, fetchOwnedShoppingLists, fetchSharedShoppingLists, deleteShoppingList, createShareLink } from 'shared';
+import { useAuth, fetchOwnedShoppingLists, fetchSharedShoppingLists, deleteShoppingList, createShareLink, AuthContextType } from 'shared';
 import Modal from '@components/Modal';
 import StatusModal from '@components/StatusModal';
 import CustomCheckbox from '@components/CustomCheckbox';
 
 import styles from './Dashboard.module.css';
-import { RequestStatus } from '@constants/enums.ts';
+import { RequestStatus } from '@constants/enums';
```

Comment:
Move this import up to be under the component imports but above the styles imports.